nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -58,8 +61,24 @@ private AccessPathNullnessAnalysis(
       Handler handler) {
     this.nullnessPropagation =
         new AccessPathNullnessPropagation(
-            Nullness.NONNULL, methodReturnsNonNull, context, config, handler);
+            Nullness.NONNULL,
+            methodReturnsNonNull,
+            context,
+            config,
+            handler,
+            new CoreNullnessStoreInitializer());
     this.dataFlow = new DataFlow(config.assertsEnabled());
+
+    if (config.checkContract()) {
```

Comment:
Love that this is behind a config flag. So, when you get to my comments on the test cases, we can always figure out the more complicated test case behavior after landing this and seeing if enabling this option internally triggers any false positives for complicated methods with `@Contract` annotations.

Reply:
Yep!